README.md

```diff
@@ -31,6 +31,29 @@ NAME                             READY   STATUS    RESTARTS   AGE
 cass-operator-555577b9f8-zgx6j   1/1     Running   0          25h
 ```
 
+### (Optional) Loading the operator via Helm
```

Comment:
Can we push this new block down to directly above the `## Features` block? People have had trouble following our directions whenever anything was the slightest bit out of order, which any optional step kinda/sorta creates. Sorry!

In the longer form docs I'd be fine with it wherever.

Reply:
No problem.